### DISCUSSION TITLE
I cannot log into my main account with 2-step authentication, nor with recovery codes, PT/EN

### DISCUSSION THREAD

**Discussion by ghost** [community/community#22082] [New to GitHub]
<p>EN:<br>
I recently migrated from a two-step authentication application, using Google’s solution, and now Microsoft’s solution. As I didn’t know how to export the data, I simply entered my profiles from the computer and migrated through the security tab to the microsoft solution.<br>
I recently formatted my pc completely, and when I went to log in to github by the main account he said that the verification code was wrong, so I decided to use the recovery code, however when I used them I was redirected back to the login screen requesting email and password, and after entering the code again. And so we were in that eternal loop until the ip was blocked. I would like to know if anyone has been through this and if you could help me, please get in touch:</p>
<p>Contato/Contact:<br>
Hotmail: <a href="mailto:f.alafyrsilva@gmail.com">f.alafyrsilva@gmail.com</a><br>
or<br>
Hotmail: <a href="mailto:f.alafyrsilva@hotmail.com">f.alafyrsilva@hotmail.com</a><br>
preferably by both.</p>
<p>PT-BR:<br>
Bem recentemente migrei de aplicativo de autenticação de duas etapas, usava há solução do google, e agora a da microsoft. Como não sabia exportar os dados simplesmente entrei nos meus perfis apartir do computador e migre pela aba segurança para a solução da microsoft.<br>
Recentemente formatei meu pc completamente, e quando fui me logar no github pela conta principal ele dizia que o código de verificação estava errado, portanto decidi usar o código de recuperação, entre tanto quando os usava era redirecionado novamente para a tela de login requisitando email e senha, e após novamente a entrada do código. E assim ficavamos nesse loop eterno até o ip ser bloqueado. gostaria de saber se alguém já passou por isso e se poderia me ajudar, por favor entre em contato acima:</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>:wave: Welcome to the community!</p>
<p>If you’re having 2FA problems and you’re unable to complete the recovery steps, please contact support directly:</p>
<p><a href="https://support.github.com/contact" class="onebox" target="_blank">https://support.github.com/contact</a></p>
